src/plugins/data/server/search/es_search/es_search_strategy.ts

```diff
@@ -30,15 +37,19 @@ export const esSearchStrategyProvider = (
     }
 
     const search = async () => {
-      const config = await config$.pipe(first()).toPromise();
-      const params = {
-        ...(await getDefaultSearchParams(uiSettingsClient)),
-        ...getShardTimeout(config),
-        ...request.params,
-      };
-      const promise = esClient.asCurrentUser.search<SearchResponse<unknown>>(params);
-      const { body } = await shimAbortSignal(promise, abortSignal);
-      return toKibanaSearchResponse(body);
+      try {
+        const config = await config$.pipe(first()).toPromise();
+        const params = {
+          ...(await getDefaultSearchParams(uiSettingsClient)),
+          ...getShardTimeout(config),
+          ...request.params,
+        };
+        const promise = esClient.asCurrentUser.search<SearchResponse<unknown>>(params);
+        const { body } = await shimAbortSignal(promise, abortSignal);
+        return toKibanaSearchResponse(body);
+      } catch (e: any) {
```

Comment:
Would it be possible to remove this `any` and other `any`s used inside catch clause when rethrowing? 
afaik we trying to double down on reducing `any` usage across codebase especially in non-test code and seems like just `Error` here should work here? 
or use `unknown` then and do all possible runtime safety checks inside`getKbnServerError` making it handling unknowns